protected/humhub/modules/admin/views/information/database.php

```diff
@@ -1,8 +1,27 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
```

Comment:
Should change this to `2018` to keep with the times. :wink:

Reply:
Ref https://github.com/humhub/humhub/issues/3058

Reply:
This file was created much earlier :/

Reply:
Hmm... Then wouldn't it be better to use `2017-2018`? I remember seeing something along the lines of being used, either way, is fine, just my thoughts on the whole copyright signing. :thinking: